nservicebus/gateway/index.md

```diff
@@ -8,49 +8,46 @@ related:
+Looking at this scenario from a logical point of view, all the pricing communication goes on within the same business service (BS). The different physical sites have different logical behavior. This is a sure sign that the gateway might come in handy. Dig deeper and look at the actual responsibilities of each site:
 
  * Headquarters - Maintains the prices and pushes any price change out to the different stores on a daily basis
  * Store - Stores the prices locally for read-only purposes
 
-Prices are usually set for at least a day at a time so it's good enough for the HQ to push them to the sites once per day. Model this as `DailyPriceUpdatesMessage` containing the list of price updates for the coming business day. Given this design, you only need to get one message across to each site per day, which lowers the requirement for the infrastructure.
+Prices are usually set for at least a day at a time so it's good enough for the HQ to push them to the sites once per day. Model this as `DailyPriceUpdatesMessage` containing the list of price updates for the coming business day. Given this design, only one message is required for each site per day, which lowers the requirement for the infrastructure.
 
-Internally in HQ, other business services may need more frequent updates, so model this with another logically different message, `PriceUpdatedForProduct`, which allows you to use the pub/sub pattern while communicating with other BS.
+Internally in HQ, other business services may need more frequent updates, so model this with another logically different message, `PriceUpdatedForProduct`, which allows the use of the (pub/sub pattern)[/nservicebus/messaging/publish-subscribe] while communicating with other BS.
 
-The gateway doesn't support pub/sub (more on that later) but this isn't a problem since request/response is perfectly fine within a BS, remembering that those sites are physically different but the communication is within the same logical BS. So when you use the gateway, the guideline is to model the messages going explicitly across sites. If you extend the sample to include a sales service responsible for reporting the sales statistics so that the pricing service can set appropriate prices, you get the following picture:
+The gateway doesn't support pub/sub (more on that later) but this isn't a problem since request/response is perfectly fine within a BS, remembering that those sites are physically different but the communication is within the same logical BS. So when using the gateway, the guideline is to model the messages going explicitly across sites. The following picture illustrates When extending the sample to include a sales service responsible for reporting the sales statistics so that the pricing service can set appropriate prices, the following picture:
```

Comment:
The last sentence contains the phrase "The following picture" twice. Once at the end and once at the beginning